data/tabular/qm9/meta.yaml

```
@@ -0,0 +1,192 @@
+---
+name: qm9
+description: |-
+    QM9 is a comprehensive dataset that provides geometric, energetic,
+    electronic and thermodynamic properties for a subset of GDB-17
+    database, comprising 134 thousand stable organic molecules with up
+    to 9 heavy atoms.  All molecules are modeled using density
+    functional theory (B3LYP/6-31G(2df,p) based DFT).
+targets:
+    - id: rotational_constant_a
+      description: Rotational constant A or the moment of inertia of the molecule along its principal axis of rotation
+      units: GHz
+      type: float
+      names:
+          - noun: Rotational constant A
+          - noun: moment of inertia along principal axis of rotation
+    - id: rotational_constant_b
+      description: Rotational constant B or the moment of inertia of the molecule along an axis perpendicular to the principal axis
+      units: GHz
+      type: float
+      names:
+          - noun: Rotational constant B
+          - noun: moment of inertia along an axis perpendicular to the principal axis
+    - id: rotational_constant_c
+      description: Rotational constant C or the moment of inertia of the molecule along an axis perpendicular to the principal axis
+      units: GHz
+      type: float
+      names:
+          - noun: Rotational constant C
+          - noun: moment of inertia along the third perpendicular to the principal axis
+    - id: dipole_moment
+      description: dipole moment
+      units: Debye
+      type: float
+      names:
+          - noun: dipole moment
+          - noun: mu
+    - id: polarizability
+      description: isotropic polarizability
+      units: Bohr^3
+      type: float
+      names:
+          - noun: isotropic polarizability
+          - noun: alpha
+    - id: homo
+      description: energy of Highest occupied molecular orbital (HOMO)
```

```suggestion
      description: energy of Highest Occupied Molecular Orbital (HOMO)
```

we should do either all caps to match the abbreviation or all low caps